<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="accessibleName">
   <string>Gestion de partenaires</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabajout">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>991</width>
      <height>571</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string> Partenaires</string>
     </attribute>
     <widget class="QLabel" name="etid">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>identifiant :</string>
      </property>
     </widget>
     <widget class="QLabel" name="etentreprise">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom d'entreprise :</string>
      </property>
     </widget>
     <widget class="QLabel" name="etservice">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>service :</string>
      </property>
     </widget>
     <widget class="QLabel" name="etnom">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom :</string>
      </property>
     </widget>
     <widget class="QLabel" name="etprenom">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom :</string>
      </property>
     </widget>
     <widget class="QLabel" name="etmail">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Email :</string>
      </property>
     </widget>
     <widget class="QLabel" name="etnumeros">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Numeros :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edid">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="edentremprise">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ednom">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="edservice">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>180</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="edemail">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>230</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ednumeros">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>280</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="edprenom">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addpart">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>360</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>682</x>
        <y>357</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Consulter la liste</string>
     </attribute>
     <widget class="QTableView" name="tableau">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>691</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="edrecherche">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>320</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnrecherche">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>320</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Recherche</string>
      </property>
     </widget>
     <widget class="QTableView" name="tablerech">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>370</y>
        <width>801</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="idetsupp_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>330</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>NOM:</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>90</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ascendant</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>130</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>descendant</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>41</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>id</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>nomentr</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>prenom</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pdf">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>210</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Mailling</string>
     </attribute>
     <widget class="QLineEdit" name="rcpt">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>501</width>
        <height>22</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>destinataire</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="msg">
      <property name="geometry">
       <rect>
        <x>23</x>
        <y>136</y>
        <width>801</width>
        <height>301</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>ecriver votre message.....</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="file">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>591</width>
        <height>22</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>attachement file</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mail_pass">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>20</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>votre mot de passe email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="subject">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>591</width>
        <height>22</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>objet(s)/subject</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browsebtn">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>100</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>browser</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sendbtn">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>440</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>envoie</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Mailling2</string>
     </attribute>
     <widget class="QAxWidget" name="WebBrowser">
      <property name="control" stdset="0">
       <string>{8856f961-340a-11d0-a96b-00c04fd705a2}</string>
      </property>
      <property name="geometry" stdset="0">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>971</width>
        <height>521</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>map</string>
     </attribute>
     <widget class="QAxWidget" name="axWidget">
      <property name="control" stdset="0">
       <string>{8856f961-340a-11d0-a96b-00c04fd705a2}</string>
      </property>
      <property name="geometry" stdset="0">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>831</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Eliminer un partenaire</string>
     </attribute>
     <widget class="QLabel" name="idetsupp">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ID :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="supid">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="supp">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>150</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1046</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QAxWidget</class>
   <extends>QWidget</extends>
   <header>qaxwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
